### PR TITLE
Finished the create functionality for events, few minor teaks to the …

### PR DESCRIPTION
**Pull request by jon-troupe**
…seed

### CODE REVIEW

File: app/views/events/new.html.erb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <%= simple_form_for(@event) do |f| %>
   <%= f.input :date, as: :date, html5: true %>
+  <%= f.input :status, collection: ["Open", "Closed"] %>
```
Comment (laura-gonzalez-z) [inline app/views/events/new.html.erb]: I think this could use `collection: Event::STATUS`

File: db/seeds.rb [hunk 2]
```diff
@@ -125,7 +125,7 @@
 5.times do
   Event.create!(
     date: DateTime.now,
-    status: "Open",
+    status: ["Open", "Closed"].sample,
```
Comment (laura-gonzalez-z) [inline db/seeds.rb]: In all of these we can also use Event::STATUS.sample

**Review by laura-gonzalez-z — COMMENTED** [review]
I'm not sure about the use case for open and closed, and we need to validate it with the group. I think it's just to enable guests (?)

**Comment by jon-troupe** [comment]
Used Event::STATUS as per Laura's suggestions.

**Comment by SaneCashew** [comment]
> 

I'm not 100% certain what Ricky has in mind, but my working assumptions is that open = anyone can come and closed = invite only to enable guests, as you said.

**Comment by jon-troupe** [comment]
> I'm not sure about the use case for open and closed, and we need to validate it with the group. I think it's just to enable guests (?)

I'm not 100% certain what Ricky has in mind, but my working assumptions is that open = anyone can come and closed = invite only to enable guests, as you said.

**Pull request merged**